src/components/AnnotationSettings.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Switch from '@material-ui/core/Switch';
+
+/**
+ * AnnotationSettings is a component to handle various annotation
+ * display settings in the Annotation companion window
+*/
+export class AnnotationSettings extends Component {
+  /**
+   * Returns the rendered component
+  */
+  render() {
+    const { displayAll, t, toggleAnnotationDisplay } = this.props;
+
+    return (
+      <FormControlLabel
+        control={(
+          <Switch
+            checked={displayAll}
+            onChange={toggleAnnotationDisplay}
+            value={displayAll ? 'all' : 'select'}
```

Comment:
I wonder if we should add `disabled={true}` if there is only 1 annotation in total.

Reply:
This is a setting that apples to all canvases, so I'm not sure that would be desirable.

Reply:
I suppose it could switch between disabled/enabled when paging through canvases, but this wasn't specified anywhere in the designs AFAICT.  @jvine @ggeisler am I missing a requirement about disabling the toggle?

Reply:
@ggeisler what do you think? Making the control wink on and off as the user pages through canvases with varying numbers of annotations might be distracting. But if there are no annotations for a canvas, it might be odd to have a control that does nothing visible on this canvas but does affect what you'll see on another canvas.

Reply:
Yeah, there wasn't a design for this specifically because the annotation mockups were only a first-pass at the general potential features.

My inclination is to go with the disabled-when-appropriate behavior, though I see the potential distraction it might cause when paging through canvases. But maybe that situation won't come up all that often, and maybe when it does won't be as distracting as it seems it might, and it seems better to always have logical behavior for the control.

Reply:
Now disabling for `0` or `1` annotations on a canvas.